File: Java Fundamentals/BasicSyntaxCondStatementsLoops/src/PrintAndSum.java
import java.util.Scanner;

public class PrintAndSum {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        int firstNum = Integer.parseInt(scan.nextLine());
        int secondNum = Integer.parseInt(scan.nextLine());
        int sum = 0;

        for (int i = firstNum; i <= secondNum ; i++) {
            sum += i;
            System.out.printf(i + " ");
        }

        System.out.printf("%nSum: %d",sum);
    }
}

File: JS Front-End/JS Syntax Fundamentals/PrintAndSum.js
function printAndSum(firstNum, secondNum){
    if(firstNum > secondNum){
        let temp = firstNum;
        firstNum = secondNum;
        secondNum = temp;
    }

    let sum = 0;
    let numLine = ""
    while(firstNum <= secondNum){
        sum += firstNum;
        numLine += (firstNum + " ");
        firstNum++;
    }

    console.log(numLine);
    console.log("Sum: " + sum);
}

printAndSum(5, 10);
printAndSum(0, 26);
printAndSum(50, 60);